lib/bloc/sentiment/sentiment_bloc.dart

```diff
@@ -22,11 +24,21 @@ class SentimentBloc extends Bloc<SentimentEvent, SentimentState> {
     try {
       emit(LoadingSentimentState());
       final positiveSentiment = await _sentimentRepo.getSentiment(event.query);
+      final negativeSentiment = positiveSentiment;
```

Comment:
I am not sure what is the reason of this line? It looks like it's only purpose is to check for nullability on line 37 which I think is unnecessary. It could be removed